src/slider.test.events.multiple.ts

```diff
@@ -0,0 +1,896 @@
+import sinon from 'sinon';
+import { sendKeys } from '@web/test-runner-commands';
+import Slider from './slider.js';
+
+it('dispatches an "input" event typing in the minimum input', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('input', spy);
+
+  const minimumInput = host.shadowRoot?.querySelector<HTMLInputElement>(
+    '[data-test="minimum-input"]',
+  );
+
+  minimumInput?.focus();
+
+  await sendKeys({ type: '1' });
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches an "input" event typing in the maximum input', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('input', spy);
+
+  const maximumInput = host.shadowRoot?.querySelector<HTMLInputElement>(
+    '[data-test="maximum-input"]',
+  );
+
+  maximumInput?.focus();
+
+  await sendKeys({ type: '1' });
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches a "change" event blurring the minimum input', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('change', spy);
+
+  const minimumInput = host.shadowRoot?.querySelector<HTMLInputElement>(
+    '[data-test="minimum-input"]',
+  );
+
+  minimumInput?.focus();
+
+  await sendKeys({ type: '1' });
+
+  expect(spy.callCount).to.equal(0);
+
+  minimumInput?.blur();
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches a "change" event blurring the maximum input', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('change', spy);
+
+  const maximumInput = host.shadowRoot?.querySelector<HTMLInputElement>(
+    '[data-test="maximum-input"]',
+  );
+
+  maximumInput?.focus();
+
+  await sendKeys({ type: '1' });
+
+  expect(spy.callCount).to.equal(0);
+
+  maximumInput?.blur();
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches an "input" event dragging the minimum handle', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('input', spy);
+
+  const sliderTrack = host.shadowRoot?.querySelector('[data-test="slider"]');
+
+  const minimumHandle = host.shadowRoot?.querySelector(
+    '[data-test="minimum-handle"]',
+  );
+
+  const trackRect = sliderTrack?.getBoundingClientRect();
+
+  assert(trackRect);
+  assert(minimumHandle);
+
+  minimumHandle.dispatchEvent(
+    new MouseEvent('mousedown', {
+      bubbles: true,
+      cancelable: true,
+    }),
+  );
+
+  await aTimeout(0);
```

Comment:
Wanna say why here and below?

Reply:
Thanks for pointing this out - turns out in most cases we only care about `await host.updateComplete`. There are only two edge case tests ( `prevents track click events dispatching immediately after completing a drag operation` and `prevents track click events dispatching immediately after completing a drag operation` ) where we do need `await aTimeout(0)` and those two areas have comments. Thank you!